android/src/main/java/com/robinpowered/react/vitals/RNVitalsModule.java

```diff
@@ -0,0 +1,111 @@
+package com.robinpowered.react.vitals;
+
+import android.os.Environment;
+import android.os.StatFs;
+import android.os.Build;
+import android.os.Debug;
+import android.content.ComponentCallbacks;
+import android.content.res.Configuration;
+import android.app.ActivityManager;
+import android.content.Context;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+import java.util.Map;
+import java.util.HashMap;
+import javax.annotation.Nullable;
+
+import java.io.File;
+
+public class RNVitalsModule extends ReactContextBaseJavaModule implements ComponentCallbacks {
+
+  public static final String MODULE_NAME = "RNVitals";
+  public static final String LOW_MEMORY = "LOW_MEMORY";
+
+  public RNVitalsModule(ReactApplicationContext reactContext) {
+    super(reactContext);
+  }
+
+  private double toMB(long num) {
+    return (double) num / 1024 / 1024;
```

Comment:
`(double) (num / 1024 / 1024)` 

to make it clear that it's not doing `((double) num) / 1024 / 1024)`